src/commands/apps-android-sha-list.ts

```diff
@@ -0,0 +1,59 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+import Table = require("cli-table");
+
+import { Command } from "../command";
+import * as getProjectId from "../getProjectId";
+import { listAppAndroidSha, AppAndroidShaData } from "../management/apps";
+import { requireAuth } from "../requireAuth";
+import * as logger from "../logger";
+
+function logCertificatesList(certificates: AppAndroidShaData[]): void {
+  if (certificates.length === 0) {
+    logger.info(clc.bold("No SHA certificate hashes found."));
+    return;
+  }
+  const tableHead = ["App Id", "SHA Id", "SHA Hash", "SHA Hash Type"];
+  const table = new Table({ head: tableHead, style: { head: ["green"] } });
+  certificates.forEach(({ name, shaHash, certType }) => {
+    const splitted = name.split("/");
```

Comment:
Can you add a comment here explaining the format of `name` so that future developers know why you're splitting it and taking the 3rd and 5th segments?